connect/docs/advanced-topics.md

```diff
@@ -1,12 +1,11 @@
 # Advanced Topics
 
-- Easy Testing
-  - example: any
```

Comment:
Instead of just deleting it you could change it to a link and a quick description

Reply:
at the moment, I am just using this file as a to-do list. Once the pages are done, I will build something out on this page.